inst/examples/02-components.Rmd

```diff
@@ -230,11 +230,21 @@ Theorems\index{theorem} and proofs are commonly used in articles and books in ma
 In **bookdown**, the types of theorem environments supported are in Table \@ref(tab:theorem-envs). To write a theorem, you can use the syntax below:
 
 ````markdown
-`r ''````{theorem}
-Here is my theorem.
-```
+::: theorem
+This is a `theorem` environment which can contain **any** _markdown_ syntax
+:::
 ````
 
+or the longer version
```

Comment:
I think it is very likely that users need other attributes in `{}`, so I tend not to introduce the shorter version `::: theorem` specifically. The longer version is not significantly longer (three more characters), and users will probably always use this syntax. I hope it makes sense to introduce only one syntax here. Otherwise we'll have to repeatedly remind readers that we are using the short or long syntax later.

Reply:
I hesitated about that so I understand the comment. The short version is nice when you do not have any attributes or other class. But yes it may not be useful to introduce two versions, you're right.

Reply:
My experience is that when people write a theorem, they are most likely to reference it somewhere (this is too common in math/stats/theory papers), which means they will need to assign an ID to it, hence they will need at least one attribute.